### PR TITLE
Change log ack from all logs to merge commit log

### PR DESCRIPTION
**Pull request by sbe-arg**
Solves https://github.com/anothrNick/github-tag-action/issues/194

I personally think this should be the expected behaviour when we merge we define the future of this branch bump in the merge commit message not necessarily by a random commit lost down in the commit history making it hard to track.

### CODE REVIEW

**Review by sammcj — APPROVED** [review]

**Comment by sbe-arg** [comment]
Moving to draft as I whink there is a bug in the tests. Triying to fix

**Comment by sbe-arg** [comment]
@sammcj have a look I have made some changes in the tests to contemplate different bumps https://github.com/anothrNick/github-tag-action/actions/runs/3177807478

File: README.md [hunk 1]
```diff
@@ -47,6 +76,7 @@ _NOTE: set the fetch-depth for `actions/checkout@v2` or newer to be sure you ret
 - **DRY_RUN** _(optional)_ - Determine the next version without tagging the branch. The workflow can use the outputs `new_tag` and `tag` in subsequent steps. Possible values are `true` and `false` (default).
 - **INITIAL_VERSION** _(optional)_ - Set initial version before bump. Default `0.0.0`.
 - **TAG_CONTEXT** _(optional)_ - Set the context of the previous tag. Possible values are `repo` (default) or `branch`.
+- **PRERELEASE** _(optional)_ - Define if workflow runs on prerelease flag or not, `false` by default. Note this will be overwritten if using complex suffix release branches.
```
Comment (sammcj) [inline README.md:79]: By prerelease flag - do you mean prerelease _tag_ perhaps?

Comment (sbe-arg) [reply README.md:79]: Yes Ill re write that a bit

**Review by sammcj — APPROVED** [review]

**Pull request merged**